tests/test_SequenceManager.cc

```diff
@@ -520,12 +524,279 @@ TEST_CASE("SequenceManager: store_sequence() & load_sequence() - Empty sequence"
+        REQUIRE(expected_msg == last_msg);
+
+
+        // use raw repository to access status
+        const auto stats = repo.status();
+        REQUIRE(stats.size() != 0);
+        bool seq_exists = false;
+        for(const auto& elm: stats)
+        {
+            if (gul14::starts_with(elm.path_name, "git_sequence_1"))
+            {
+                seq_exists = true;
+
+                // if staging did not work: 
+                //      elm.handling == "untracked"
+                //      elm.changes == "untracked"
+                // if commit did not work
+                //      elm.handling == "staged"
+                //      elm.changes == "new file"
+                REQUIRE(elm.handling == "unchanged");
+                REQUIRE(elm.changes == "unchanged");
+            }
+        }
+        // check if created sequence is at least indexed by git
+        REQUIRE(seq_exists);
+    }
+
+    SECTION("change and store sequence (step_setup)")
+    {
+        SequenceManager manager{ git_dir };
+
+        // load sequence an change step setup
+        auto seq = manager.load_sequence(UniqueId{ 0xabcdef123456 });
+        const auto step_setup_script =
+        R"(
+        a = 'Bob'
+        function test(name)
+            return name .. ' with some funky stuff!'
+        end
+        b = test('Alice') \t\b\b  c = 4)";
+        seq.set_step_setup_script(step_setup_script);
+
+        // store sequence
+        manager.store_sequence(seq);
+
+        git::GitRepository repo{ git_dir };
+
+        const std::string expected_msg = "Modify sequence git_sequence_1[0000abcdef123456]\n\n"
+            "- modified: sequence.lua";
+        const auto last_msg = repo.get_last_commit_message();
+        REQUIRE(expected_msg == last_msg);
+
+        // use raw repository to access status
+        const auto stats = repo.status();
+        REQUIRE(stats.size() != 0);
+        bool seq_exists = false;
+        for(const auto& elm: stats)
+        {
+            if (gul14::starts_with(elm.path_name, "git_sequence_1[0000abcdef123456]/sequence.lua"))
+            {
+                seq_exists = true;
+
+                // if staging did not work: 
+                //      elm.handling == "unstaged"
+                //      elm.changes == "modified"
+                // if commit did not work
+                //      elm.handling == "staged"
+                //      elm.changes == "modified"
+                REQUIRE(elm.handling == "unchanged");
+                REQUIRE(elm.changes == "unchanged");
+            }
+        }
+        // check if created sequence is at least indexed by git
+        REQUIRE(seq_exists);
+    }
+
+    SECTION("copy sequence")
+    {
+        SequenceManager manager{ git_dir };
+
+        // find sequence unique ID
+        UniqueId uID{ 0 };
+        for (auto elm: manager.list_sequences())
+        {
+            if (elm.name == SequenceName{ "git_sequence_1" }) uID = elm.unique_id;
+        }
```

Comment:
Could be a `std::find_if()` or even a helper function (the pattern repeats several times).

Reply:
Same for libgit4cpp tests, linking
* https://github.com/taskolib/libgit4cpp/issues/12

Reply:
Done.